eduvpn/actions/select.py

```diff
@@ -71,10 +72,6 @@ def select_profile(builder, verifier, lets_connect):
             twofa_label.set_text("")
             twofa_label_label.set_text("")
 
-        if are_we_running_ubuntu1804():
```

Comment:
Ook hier, dit is een andere issue. Op zich goed dat je dit ook al oplost, maar laten we proberen een PR single issue te houden. Dan kunnen we ze eerder afronden.